reproducibility_FAQs.md

```diff
@@ -0,0 +1,253 @@
+- **Exhibit-by-exhibit status** noting whether each table or figure in the paper was successfully reproduced
+
+
+### What are the most common reasons reproducibility checks fail?
+
+Even well-documented research packages can fail reproducibility checks. Below are the most frequent issues we encounter:
+
+- **Unstable Code (Results change with every run):**  
+  Reproducibility requires that the same inputs always produce the same outputs. If tables or figures change between runs, the code is unstable. This often happens due to unsorted merges, random processes without a set seed, or non-unique identifiers used for sorting. Instability makes it impossible to confidently verify findings.
+  Most reproducibility issues are solved after setting a random seed in the beginning of the code. Some other issues can occur in Stata, which are explained in [this presentation](https://osf.io/2bxn7).
+
+- **Starts from Intermediate Data:**  
+  The package should begin with the original/raw data (see the question below on what is original data for more information). If it starts from partially processed or undocumented intermediate datasets, it becomes unclear how those datasets were constructed, preventing full reproducibility. 
+
+- **Manual Steps Not Documented:**  
+  If figures or tables are modified manually (e.g., in Excel) without clear instructions or supporting code, the process becomes opaque. All steps used to generate results—even if manual—must be documented clearly in the README or equivalent instructions.
+
+- **Coding Errors:**  
+  Broken scripts, incorrect file paths, missing dependencies, or syntax errors can prevent the code from running altogether. Ensure to run the code once before submitting the package.
+
+- *Incomplete Version Control or Environment Info:**  
+  Failing to specify software versions (e.g., using Stata 18 but not declaring it), or not locking Python/R environments (e.g., with `requirements.txt`, `renv.lock`) can cause code to behave differently on other systems. Reproducibility requires knowing exactly what software environment was used. [This presentation](https://osf.io/q7rjt) contains more information on this issue and potential solutions.
+
+- **Manuscript Version Not Aligned with Code:**  
+  In some cases, the code is up to date but the manuscript is not. Or vice versa—the manuscript reflects newer analyses, but the package reflects an older draft. This creates discrepancies between exhibits and outputs. Always ensure that the final manuscript matches the results generated by the package.
+
+
+### What if my package uses Excel?
+
+While Excel is widely used, reproducibility verification is  **much more difficult**  when workflows rely on manual steps, hidden formulas, or undocumented edits. Therefore,  **we strongly discourage using Excel for critical parts of the analysis**  and recommend automating as much of the workflow as possible.
```

Comment:
I'm not sure the bold is warranted here ... yes, excel is a problem, but singling it out for bold feels a little much (i feel as strongly about version control problems!). the language is fine, but i recommend removing (or at least reducing) the bold.